packages/grafana-data/src/utils/unaryOperators.ts

```diff
@@ -0,0 +1,45 @@
+import { Registry, RegistryItem } from './Registry';
+
+export enum UnaryOperationID {
+  Abs = 'abs',
+  Exp = 'exp',
+  Ln = 'ln',
+  Floor = 'floor',
+  Ceil = 'ceil',
+}
+
+export type UnaryOperation = (value: number) => number;
+
+interface UnaryOperatorInfo extends RegistryItem {
+  operation: UnaryOperation;
+}
+
+export const unaryOperators = new Registry<UnaryOperatorInfo>(() => {
+  return [
+    {
+      id: UnaryOperationID.Abs,
+      name: 'abs',
```

Comment:
I think for the names we should do the full name as opposed to underlying JS function name. i.e. use "Absolute Value", "Natural Expopent (e^x)", etc.

Reply:
I assume you mean to change these in the UI also. I kept only the function name to be in line with how the binary operations work (e.g.: mimic the operation (field + field, abs(field)). I think having the docs updated to point out what each operation means is enough.